docs/concepts/comparisons.md

```diff
@@ -0,0 +1,35 @@
+---
+title: 'IPFS Comparisons'
+description: Comparing IPFS to other solutions.
+---
+
+# Storage Solution Comparisons
+
+## How is IPFS different from other solutions?
+
+IPFS is a general-purpose file system that uses a distributed hash table (DHT) to store and retrieve data. This sets it apart from other solutions with a more specific focus or using different data storage mechanisms. For example:
```

Comment:
Suggestion: This sentence 
```
This sets it apart from other solutions with a more specific focus or using different data storage mechanisms. For example:
```
reads a little strange. I think rephrasing it to something like
```
This sets it apart from other storage solutions that use different data storage mechanisms or have a more specific focus .  Similar solutions include:
```
would read better.